src/store/participantsStore.js

```diff
@@ -321,27 +321,29 @@ const mutations = {
 			if (state.inCall[token] && state.inCall[token][sessionId]) {
 				Vue.delete(state.inCall[token], sessionId)
 			}
-
-			if (state.connecting[token] && state.connecting[token][sessionId]) {
```

Comment:
I think we can keep this resetting (in case it survives somehow). Maybe via `commit('finishedConnecting')`?